docs/workspaces.md

```diff
@@ -46,20 +46,20 @@ can be defined on a per-workspace basis to automate the workspace start/stop.
 
 ![Scheduling UI](./images/schedule.png)
 
-### Auto-start
+### Autostart
 
-The auto-start feature automates the workspace build at a user-specified time
+The autostart feature automates the workspace build at a user-specified time
 and day(s) of the week. In addition, users can select their preferred timezone.
 
-![Auto-start UI](./images/auto-start.png)
+![Autostart UI](./images/autostart.png)
```

Comment:
This will break existing links unless you also rename the corresponding files.

Reply:
true